lib/sqeduler/middleware/kill_switch.rb

```diff
@@ -0,0 +1,30 @@
+module Sqeduler
+  module Middleware
+    # Verifies that worker classes are enabled before enqueuing a job.
```

Comment:
Oh, never mind, I see now this is meant to be used as a client-side middleware. Maybe worth clarifying in the comment.